lib/phoenix_live_view/router.ex

```diff
@@ -73,10 +73,14 @@ defmodule Phoenix.LiveView.Router do
       module
       |> Atom.to_string()
       |> String.split(".")
-      |> Enum.take(2)
+      |> remove_last()
       |> Kernel.++(["LayoutView"])
       |> Module.concat()
 
     {view, :app}
   end
+
+  defp remove_last(module_parts) do
+    Enum.take(module_parts, length(module_parts) - 1)
```

Comment:
```suggestion
    Enum.drop(module_parts, -1)
```

Reply:
We can also inline this without the private function

Reply:
Updated! 👍